docs/docker-compose.md

```diff
@@ -189,6 +189,105 @@ volumes:
+            containerHttpPort: $$cap_OPENVPN_PORT_ADMIN
+        image: linuxserver/openvpn-as:$$cap_OPENVPN_VERSION
+        hostname: $$cap_appname.$$cap_root_domain
+        environment:
+            PUID: $$cap_OPENVPN_PUID
+            PGID: $$cap_OPENVPN_GUID
+            TZ: $$cap_OPENVPN_TZ
+            INTERFACE: $$cap_OPENVPN_INTERFACE
+        volumes:
+            - $$cap_appname:/config
+        ports:
+            - $$cap_OPENVPN_PORT_TCP:$$cap_OPENVPN_PORT_TCP
+            - $$cap_OPENVPN_PORT_UDP:$$cap_OPENVPN_PORT_UDP
+        cap_add:
+            - NET_ADMIN
+
+caproverOneClickApp:
+    displayName: OpenVPN Access Server
+    isOfficial: false
+    description: Full featured secure network tunneling VPN software.
+    documentation: https://openvpn.net/index.php/access-server/overview.html
+    instructions:
+        start: |-
+            OpenVPN is a full featured, open-source VPN solution that accommodates a wide range of configurations.
+
+            **Important notes:**
+            - The provider of this image (LinuxServer) has decided to **deprecate** it in favor of WireGuard
+            - You need **access via SSH** to your server after deployment to modify a file (only once)
+            - You may need to **open the ports** `1194` and `9443` on your firewall
+        end: |-
+            OpenVPN has been deployed **successfully**!
+
+            Before you can start using it, you need to perform some steps.
+            You only need to do this once.
+            **Please screenshot/copy the following instructions before proceeding:**
+
+            1. Go to the **HTTP Settings** of your deployment `$$cap_appname` here in CapRover
+            2. **Enable HTTPS**
+            3. **Edit Default Nginx Configurations** and search for the line `set $upstream http://<%-s.localDomain%>:<%-s.containerHttpPort%>;` and replace `http://` with `https://`
+            4. Go to https://$$cap_appname.$$cap_root_domain/admin and log in using user `admin` and password `password`
+            5. Navigate to **Configuration** > **Network Settings** and replace the value in **Hostname Or IP Address** with `$$cap_appname.$$cap_root_domain` and update the server (if you get an error just refresh the page, log back in to the admin page (`/admin`) and proceed with the next step)
+            6. Navigate to **User Management** > **User Permissions** and add **New Username**, set a password, make it an admin and **Save Settings**.
+            7. Log out and log in as the new user
+            8. Navigate to **User Management** > **User Permissions** and delete the `admin` account
+            9. Access via SSH to your server and execute `docker exec -it $(docker ps --filter name=srv-captain--$$cap_appname -q) sed -i '/boot_pam_users.0=admin/c\boot_pam_users.0=abcnonexistentuserxyz' /config/etc/as.conf` to block the `admin` account and prevent its creation during container restart
+
+            Enjoy your private VPN!
+    variables:
```

Comment:
Similar to above, `variables` section can be removed completely from this example. Just replace them inline for a simpler and less confusing example. (See the wordpress example on this page)

Reply:
I was trying to find an example that make sense. No one will run CAP_NET_ADMIN on a wordpress. For a VPN it can make sense. I will try to do something less heavy than still makes sense

Reply:
Totally! You should definitely use the VPN example. What I am saying is that we should get rid of the fluff (variables, instructions, etc), so that it's obvious where it is being used.